packages/compass-database/src/components/database/database.jsx

```diff
@@ -56,15 +54,15 @@ class Database extends Component {
    */
   render() {
     return (
-      <div className={classnames(styles.database)}>
+      <div className={styles.database}>
         <TabNavBar
```

Comment:
I think the tab form is a bit more constrained in its available boundaries+scrolling since it's rendered inside of a card. I think using the `Tabs` and `Tab` in the connect form with custom styles might make more sense then binding it to sharing this. Can definitely do it though if you'd like it to share.